tests/test_cli.py

```diff
@@ -144,7 +144,7 @@ def _test_add(bdb, prompt, *, add_tags=[], tag=[], tags_fetch=True, tags_in=None
 @pytest.mark.parametrize('count', [{}, {'count': ['10']}])
 @pytest.mark.parametrize('order, indices, command', [
     (['tags', '-url'], None, {'order': ['tags,-url'], 'print': []}),
-    (['-description', '+uri'], {5, 8, 9, 10, 11, 12},
+    (['-description', '+uri'], [5, 8, 9, 10, 11, 12],
```

Comment:
`random.sample()` doesn't work on sets (though it works on ranges); so I'm converting the parsed IDs from the input to a list now.